src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  display: flex;
+  width: 200px;
+  background-color: #fff;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition-duration: 300ms;
+
+  &__image {
+    display: flex;
+    height: 134px;
+    width: 160px;
+    margin: 32px auto 0;
+  }
+
+  &__product-name {
+    display: flex;
+    font-weight: 500;
+    padding-inline: $padding;
+    color: $main-text-color;
+    margin-top: 40px;
+    font-size: 12px;
+    line-height: 18px;
+    transition-duration: 300ms;
+
+    &:hover {
+      color: #34568b;
+      cursor: default;
+    }
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    display: flex;
+    font-weight: 400;
+    color: $additional-text-color;
+    font-size: 10px;
+    padding-left: $padding;
+    line-height: 14px;
+  }
+
+  &__review {
+    display: flex;
+    padding-inline: $padding;
+    justify-content: space-between;
+    margin-top: 16px;
+    align-items: center;
+
+    @import './stars';
+
+    .card__review-quantity {
+      font-size: 10px;
+      font-weight: 400;
+      color: $main-text-color;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    padding-inline: $padding;
+    justify-content: space-between;
+    line-height: 18px;
+    align-items: center;
+    margin-top: 24px;
+
+    .card__price-title {
+      font-weight: 400;
+      font-size: 12;
```

Comment:
The font-size property is missing a unit. It should be specified, for example, as '12px'.